eclipse-plugin/eclipse/cc.codechecker.eclipse.plugin/src/cc/codechecker/plugin/config/CodeCheckerContext.java

```diff
@@ -158,10 +158,10 @@ public void setActiveEditorPart(IEditorPart partRef,boolean refresh) {
                         rlv.onEditorChanged(project, filename);
                     }
                 }
-                if (vp.getId().equals(BugPathListView.ID)) {
+                /*if (vp.getId().equals(BugPathListView.ID)) {
                     BugPathListView bplv = (BugPathListView) vp.getView(true);
                     bplv.clear();
-                }
+                }*/
```

Comment:
again, do not comment code out, just delete it.

Also I see why this change helps solving this issue, but it also has a sideeffect of reintroducing the original problem solved by it. A better condition would be adding a condition to only call the clear method when the currently active editor isn't on the list.